### PR TITLE
warn that `dotnet new -t web`

### PR DESCRIPTION
**Pull request by Rick-Anderson**
4 PR's *fixing* this to add `-t` and lots of customer comments saying you need `-t` - time to fix.

### CODE REVIEW

File: aspnetcore/getting-started.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ uid: getting-started
     cd aspnetcoreapp
     dotnet new web
     ```
+    
+    Note: If you get an error running `dotnet new web` or you must run  `dotnet new -t web`, you'll need to upgrade to the current version of .NET Core. Install [.NET Core](https://microsoft.com/net/core)
```
Comment (tdykstra) [inline aspnetcore/getting-started.md]: "or you must run" isn't clear.  How about 

If you get an error running `dotnet new web`, don't change the command to `dotnet new -t web` to get it to work because that form of the command only works with an older version of .NET Core.  Install the current version of [.NET Core](https://microsoft.com/net/core).

Could shorten by omitting "because that form of the command only works with an older version of .NET Core" but that makes the issue a little clearer.

Comment (rhires) [inline aspnetcore/getting-started.md]: A little more change for clarity? 

If you get an error running `dotnet new web`, don't change the command to `dotnet new -t web` to get it to work. The latter form of the command only works with an older version of .NET Core. Instead, install the current version of .NET Core.

Comment (Rick-Anderson) [inline aspnetcore/getting-started.md]: They don't absolutely have to upgrade .NET Core. How about something less obtrusive like:

Note: Previous versions of .NET Core required a `t` parameter, that is   `dotnet new -t web`.    `dotnet` (with no parameters)  will display the .NET Core version.

Comment (tdykstra) [inline aspnetcore/getting-started.md]: They do have to upgrade in order to follow the tutorial instructions.  This is a getting started tutorial, we should guide them to get latest if they don't have it, and I think the "Previous versions ..." text is too short, might be confusing.

Comment (Rick-Anderson) [inline aspnetcore/getting-started.md]: check the new rewrite. Latter/former are hard to follow. Also, you can complete this tutorial with the older version using the`-t` as so many have reported. We might want to encourage them to upgrade, but it's not required.
